src/SDKInstanceClient.ts

```diff
@@ -48,6 +48,10 @@ class SDKInstanceClient implements SDKInstance {
     return await this.zDAORegistryClient.listZNAs();
   }
 
+  async listZDAOs(): Promise<any> {
+    return await this.zDAORegistryClient.listZDAOs();
```

Comment:
Assign this function call to a variable and then return that variable instead. This makes debugging easier (and IMO readability a little too). In Solidity I'd say this is the way to do it because it costs additional gas to assign a variable, but here we don't have that restriction